src/tess_catalog.jl

```diff
@@ -0,0 +1,389 @@
+## ExoplanetsSysSim/src/tess_catalog.jl
+## (c) 2020 Aditya Sengupta?
+
+using DataFrames
+using CSV
+using FileIO
+
+mutable struct TESSPhysicalCatalog
```

Comment:
Similarly, we can make abstract types: PhysicalCatalogAbstract and ObsCatalogAbstract, in hopes that can help reduce duplicated code that makes it harder to maintain.